test/asmt-double-entry.js

```diff
@@ -34,8 +74,8 @@ describe('navigate to asmt and fill out asmts', function() {
             instrumentId: 26363, // Calculation Test
             segmentInterval: 'v3', // can vary this: visit1, v1, v2, v3, v4
             sourceType: '5',  // can vary this: 3, 4, 5, 6, 7, 8, 9
-            assessmentDate: '12/24/1900', // don't change this Date, or need to make sure it's unique in resume queue
-            assessmentStartTime: '10:22',
+            assessmentDate: getRandomDate(),
```

Comment:
It will work all other tests, but _might_ break `resume entry` test. For example, if the randomly generated date already existed in the resume entry table, i.e, there will be two asmts having same asmt date in the resume table, the selenium might wrongly pick the wrong asmt to resume and fail the test.  It might help if we fix the year to some weird number like 1823 and only make the month/day randomly generated? so that the asmt can be UNIQUE in the resume table.